graylog2-web-interface/src/views/components/views/ViewPropertiesModal.jsx

```diff
@@ -23,7 +24,8 @@ export default class ViewPropertiesModal extends React.Component {
     };
   }
 
-  componentWillReceiveProps(nextProps) {
+  // eslint-disable-next-line camelcase
+  UNSAFE_componentWillReceiveProps(nextProps) {
     const { title } = this.props;
     const { view } = this.state;
     if (title !== nextProps.title || !isEqual(view, nextProps.view)) {
```

Comment:
When we turn this into a functional component, we should do something better then comparing the whole view.
1. it is not very efficient
2. we do not really care about the whole view if I am not wrong